Sources/BuildServerIntegration/BuiltInBuildServerAdapter.swift

```diff
@@ -33,47 +33,47 @@ package struct BuildSystemSpec {
 
   package var kind: Kind
 
-  /// The folder that best describes the root of the project that this build system handles.
+  /// The folder that best describes the root of the project that this build server handles.
   package var projectRoot: URL
 
-  /// The main path that provides the build system configuration.
+  /// The main path that provides the build server configuration.
   package var configPath: URL
 
-  package init(kind: BuildSystemSpec.Kind, projectRoot: URL, configPath: URL) {
+  package init(kind: BuildServerSpec.Kind, projectRoot: URL, configPath: URL) {
     self.kind = kind
     self.projectRoot = projectRoot
     self.configPath = configPath
   }
 }
 
-/// A type that outwardly acts as a BSP build server and internally uses a `BuiltInBuildSystem` to satisfy the requests.
-actor BuiltInBuildSystemAdapter: QueueBasedMessageHandler {
+/// A type that outwardly acts as a BSP build server and internally uses a `BuiltInBuildServer` to satisfy the requests.
```

Comment:
It seems like `BuiltInBuildServer` really is still `BuiltInBuildSystem` right?

Reply:
No, I think `BuiltInBuildServer` makes sense because it’s essentially a BSP server that is built into SourceKit-LSP.